commandapi-platforms/commandapi-spigot/commandapi-spigot-core/src/main/java/dev/jorel/commandapi/arguments/AsyncPlayerProfileArgument.java

```diff
@@ -0,0 +1,51 @@
+package dev.jorel.commandapi.arguments;
+
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import dev.jorel.commandapi.CommandAPIBukkit;
+import dev.jorel.commandapi.CommandAPISpigot;
+import dev.jorel.commandapi.executors.CommandArguments;
+import org.bukkit.profile.PlayerProfile;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * An argument that represents the Bukkit OfflinePlayer object with asynchronous support.
+ *
+ * @since 9.7.1
```

Comment:
I think we should update these since tags? These `PlayerProfile` classes are different from the ones they replaced.